dlk/python/dlk/plugins/tf.py

```diff
@@ -487,7 +488,7 @@ def guess_node_format(input_node: Any) -> str:
             op_type = self.convert_operator(node.op_type)
             if op_type == 'Conv':
                 return out_format, [out_format, _default_w_format, 'C']
-            elif op_type in {'QTZ_binary_mean_scaling', 'QTZ_binary_channel_wise_mean_scaling'}:
+            elif op_type in ['QTZ_binary_mean_scaling', 'BinaryChannelWiseMeanScalingQuantizer']:
```

Comment:
Could you revert to `curly bracket {}` here, since this line is doing an operation for a set, not for a sequence.

Reply:
Updated on commit 2204a88 Thanks!